### ISSUE TITLE
Possible VMAF calculation issue

### ISSUE THREAD

**Issue by Extarys** [kingstefan26/alabamaEncoder#6]
I activated `print_analysis_logs` because I was curious to see more stats :D, but I noticed something strange. When the CRF changes, the vmaf score and bitrate doesn't reflect the new CRF.

I don't see any lines with more than 3 attempts., so it might be only a logging issue though (like variables not being set right or something).

I rearrange the log by chuck so it's easier to see:
```
[70] crf: 28; vmaf: 96.187366; score_error: 3.2021752709565963; bitrate: 2377 kb/s; attempt 0/10                                                                                                
[70] crf: 37; vmaf: 96.187366; score_error: 3.2021752709565963; bitrate: 2377 kb/s; attempt 1/10                                                                                                
[70] crf: 42; vmaf: 96.187366; score_error: 3.2021752709565963; bitrate: 2377 kb/s; attempt 2/10     
[...]                                                                                           
[70] Finished with crf 45; vmaf: 96.187366; score_error: 3.2021752709565963; bitrate: 2377 kb/s                                                                                                 

[72] crf: 28; vmaf: 94.768922; score_error: 0.9940820884372329; bitrate: 1415 kb/s; attempt 0/10          
[72] crf: 37; vmaf: 94.768922; score_error: 0.9940820884372329; bitrate: 1415 kb/s; attempt 1/10                                                                                                
[72] crf: 42; vmaf: 94.768922; score_error: 0.9940820884372329; bitrate: 1415 kb/s; attempt 2/10                                                                                                
[72] crf: 45; vmaf: 94.768922; score_error: 0.9940820884372329; bitrate: 1415 kb/s; attempt 3/10
[72] Finished with crf 45; vmaf: 94.768922; score_error: 0.9940820884372329; bitrate: 1415 kb/s                                                                    

[73] crf: 28; vmaf: 95.864913; score_error: 2.2315003263536246; bitrate: 1961 kb/s; attempt 0/10        
[73] crf: 37; vmaf: 95.864913; score_error: 2.2315003263536246; bitrate: 1961 kb/s; attempt 1/10                                                                                                
[73] crf: 42; vmaf: 95.864913; score_error: 2.2315003263536246; bitrate: 1961 kb/s; attempt 2/10      
[...]
[73] Finished with crf 45; vmaf: 95.864913; score_error: 2.2315003263536246; bitrate: 1961 kb/s                                                                                                 
```

**EDIT** All chunks are tried three times and it always settles to CRF 45, I've set the ("undocumented") `--crf_limits 15,50`
**EDIT 2** Same symptoms without `crf_limits`

**Comment by kingstefan26** [comment]
this is usually symptomatic of all black scenes, is it happening to all scenes? could you please check what is contained in the affected scenes?

**Comment by kingstefan26** [comment]
Thanks a lot for your investigation! the issue is probably one of the flags you used somehow preventing CRF from updating run to run in dynamic vmaf targeting (the custom flags one most likely)

**Comment by kingstefan26** [comment]
fixed in fa33fbb3

**Comment by kingstefan26** [comment]
fixed in fa33fbb3
